src/modules/commands/discord/urbandictionary.ts

```diff
@@ -0,0 +1,83 @@
+import { Command } from '@commands/CommandHandler'
+import got from 'got'
+import { isRecord } from '@app/validators/object'
+import camelcaseKeys from 'camelcase-keys'
+import discord from 'discord.js'
+
+type definition = {
+  definition: string
+  permalink: string
+  thumbsUp: number
+  soundUrls: string[]
+  author: string
+  word: string
+  defid: number
+  currentVote: string
+  writtenOn: string
+  example: string
+  thumbsDown: number
+}
+
+export default new Command<discord.Message>({
+  keyword: 'urbandictionary',
+  description: 'get the modern definition of a word or expression',
+  help: 'Usage: `{prefix}urbandictionary` - get the modern definition of a word or expression',
+  callback: async ({ args, reply, env }) => {
+    if (args.length === 0) return reply(env.command.getHelp(env.handler))
+
+    const word = args.join(' ') //Merge all the arguments for expressions
+
+    const term = word.toLowerCase()
+    const url = `http://api.urbandictionary.com/v0/define?term=${term}`
+
+    let list = (await got(url)
+      .json()
+      .then((data) => {
+        if (!isRecord(data)) {
+          return Promise.reject(new Error(`Unexpected query return type`))
+        }
+        if (data.error) {
+          // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+          return Promise.reject(new Error(`Query error: ${data.error}`))
+        }
+        if (data.errorCode) {
+          // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+          return Promise.reject(new Error(`Query error: ${data.errorCode}`))
+        }
+        const collectionListName = 'list'
+        const definitions = data[collectionListName]
+        if (!definitions) {
+          return Promise.reject(
+            new Error(`${collectionListName} not in result`),
+          )
+        }
+        if (!Array.isArray(definitions)) {
+          return Promise.reject(
+            new Error(`${collectionListName} is not an array`),
+          )
+        }
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+        return camelcaseKeys(definitions, { deep: true })
+      })) as definition[]
+
+    // eslint-disable-next-line no-useless-escape
+    const re = /\[(\w+(\s\w+)*?)\]/g
+
+    if (list.length === 0)
+      if (args.length > 1)
+        return reply(`There is no definition for the expression **${word}**.`)
+      else return reply(`There is no definition for the word **${word}**.`)
```

Comment:
The four lines above are quite difficult to read without curly braces. Please add them.